app/src/store/stores/authStore.ts

```diff
@@ -31,6 +33,80 @@ export default class AuthStore {
     Object.values(this._store.api).forEach(api => api.setCredentials(credentials));
   }
 
+  /**
+   * Convert exception to error message
+   */
+  async getErrMsg(error: string) {
+    // determine the main error message
+    const invalidPassMsg = ['expected 1 macaroon, got 0'];
+    for (const m in invalidPassMsg) {
+      const errPos = error.lastIndexOf(invalidPassMsg[m]);
+      if (error.length - invalidPassMsg[m].length == errPos) {
+        this.errors.mainErr = l('invalidPassErr');
+        break;
+      }
+    }
+
+    let walletLocked = false;
+    if (this.errors.mainErr.length == 0) {
+      const walletLockedMsg = [
+        'wallet locked, unlock it to enable full RPC access',
+        'proxy error with context auth: unknown macaroon to use',
+      ];
+      for (const m in walletLockedMsg) {
+        const errPos = error.lastIndexOf(walletLockedMsg[m]);
+        if (error.length - walletLockedMsg[m].length == errPos) {
+          walletLocked = true;
+          this.errors.mainErr = l('walletLockedErr');
+          break;
+        }
+      }
+    }
+
+    if (this.errors.mainErr.length == 0) this.errors.mainErr = l('noConnectionErr');
+
+    // get the subserver status message
+    let litStatus = null;
+    let lndStatus = null;
+    try {
+      const serverStatus = await this._store.api.lit.listSubServerStatus();
+      const subServersMap = serverStatus['subServersMap'];
```

Comment:
Since the response object returned from the RPC has nested arrays, it make the code below that parses it pretty difficult to read. As I went through this, I had to keep asking myself "what is `subServersMap[e][0]` and `litStatus[1]`?" I'd suggest first converting it into an object that's more digestible which will make the rest of the code much easier to reason about.

Here's one way to do that
```
const serverStatus = await this._store.api.lit.listSubServerStatus();
// convert the response's nested arrays to an object mapping `subServerName` -> `{ disabled, running, error }`
const status = serverStatus.subServersMap.reduce(
  (acc, [serverName, serverStatus]) => ({ ...acc, [serverName]: serverStatus }),
  {} as Record<string, SubServerStatus>,
);
```

Now when checking for a `lit` error, it can be like this below which is easier to read IMO.
```
if (status.lit?.error) {
  this.errors.litDetail = status.lit.error;
} else if (!status.lit?.running) {
  this.errors.litDetail = 'LiT is not running.';
  if (walletLocked)
    this.errors.litDetail += ' Please ensure that the wallet is unlocked.';
}
```